ToastNotification/ToastNotification.csproj

```diff
@@ -33,7 +33,7 @@ For some platforms, you may need to install platform-specific packages.
     <PackageReadmeFile>README.md</PackageReadmeFile>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Condition=" '$(TargetFramework)' != 'uap10.0.15063' " Include="Uno.UI" Version="4.0.13"/>
+    <PackageReference Condition=" '$(TargetFramework)' != 'uap10.0.15063' " Include="Uno.UI" Version="4.0.13" />
```

Comment:
Please update the package reference here too.
Note that for conditional `PackageReference` tags like this you might have to edit the `.csproj` file directly, instead of updating from the "Manage NuGet Packages" UI from Visual Studio.